src/utils/formatNumbers.ts

```diff
@@ -0,0 +1,78 @@
+/* Copied from Uniswap/v-3: https://github.com/Uniswap/v3-info/blob/master/src/utils/numbers.ts */
+import { Currency, CurrencyAmount, Price } from '@uniswap/sdk-core'
+import { DEFAULT_LOCALE } from 'constants/locales'
+
+// Convert [CurrencyAmount] to number with necessary precision for price formatting.
+export const currencyAmountToPreciseFloat = (currencyAmount: CurrencyAmount<Currency> | undefined) => {
```

Comment:
Change these to be named functions: `export function functionName(parameters) {` instead of `export const functionname = (parameters) => {`.

Reply:
done